### ISSUE TITLE
Add javadoc for all classes/methods/members

### ISSUE THREAD

**Issue by rithms** [rithms/riot-api-java#79]
A lot of classes, methods, and member variables are missing javadoc.

Acceptance Criteria:
- General form Javadocs have been added for all classes, methods, and member variables where appropriate, following the standards currently in place.

* Javadoc must be properly formatted, and include appropriate 'at-clauses' where needed.
* Methods overriding supertype methods are an exception.

**Comment by Abrasha** [comment]
Hi @taycaldwell. How much time it will take to investigate your project to add proper JavaDoc?
Note: I have never heard about your project but I would like to contribute :)

**Comment by taycaldwell** [comment]
Proper Javadoc for this project will probably require one to be familiar with the game League of Legends, and investigation outside of the library itself to determine what certain fields in the data objects actually mean (There are some fields that are undocumented by the official API that this library wraps.).

From a pure code perspective, the project should be simple to follow.

As a software engineer I've become hard-wired to estimate issues in terms of complexity rather than time itself. It will definitely take longer to write the Javadoc if you are unfamiliar with the game, however I would consider the overall complexity of the task is fairly low.

**Comment by Abrasha** [comment]
@taycaldwell you are right. I believe that I am not the right person to help with this one :(

**Comment by Pankraz01** [comment]
is there any javadoc to see in 2019?
